### DISCUSSION TITLE
Getting started Barcode / Digikey barcodes not recognized.

### DISCUSSION THREAD

**Discussion by LTcr6** [inventree/InvenTree#8267] [Q&A] [answered]
Hello I just got started and am now up running with Inventree for a few days. I have a database of already scanned Digikey barcodes that I would like to import but it seems like I will have to possible just rescan them to get Inventree to read them into my parts database. My problem is that when I scan the bar code I do see the barcode information but it does not link to the the Digikey API like I thought it would and thus I am wondering if I have missed a setup step to get the plugin running. (Would be happy to write some documentation for how to get it up and running once I get this resolved. )

Some details for whomever comes to help:
- I am running the Docker container in WSL and I am able to scan barcodes they just do not seem to get sent or get anything back from the digikey barcode plugin that comes preinstalled with Inventree.
- I have tried to use both the newer and older platform / UI and they both give the same results (Older UI actually tells me nothing can be found where as the new UI just displays data but either way they do not give me the Digikey information. )

**Comment by SchrodingersGat** [comment, marked as answer]
The current implementation (and builtin plugins) do not link into the digikey API. It reads the fields in the DigiKey barcode and attempts to link to an internal part (which already exists) using the read data.

It would be a great addition to link into the DigiKey API and potentially provide more functionality but no one has implemented that yet

**Reply by LTcr6** [reply]
So the Digikey barcode will only show up if I already have a part made with the same information in my inventory? As for the API for the Digikey barcode let me look into that a little I think it should be posable as I saw a repo or two with it just not integrated into Inventree, I will work on a proof of concept. If you have any implementation details you would like just let me know.
